cmake/cxx_flags_override.cmake

```diff
@@ -20,6 +20,13 @@ if (("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang") OR ("${CMAKE_CXX_COMPILER_ID}"
   set(CMAKE_CXX_FLAGS_MINSIZEREL_INIT "-Os")
   set(CMAKE_CXX_FLAGS_RELEASE_INIT "-O3")
   set(CMAKE_CXX_FLAGS_RELWITHDEBINFO_INIT "-O2 -g")
+
+  if (CMAKE_VERSION VERSION_LESS "3.1")
```

Comment:
@Mic92 Only consistency with other CMake code I've written. CMake implicitly dereferences variables written like this and I'm not a huge fan of this behaviour.

I was actually confusing this with another CMake issue where CMake implicitly dereferences quoted variables (which really is evil. See CMake policy CMP0054 where it was fixed).

So what you have is fine and there's no need to change it unless you want to be consistent with other code I've written.